C/plugins/south/dht11/dht11.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * FogLAMP south service plugin
+ *
+ * Copyright (c) 2018 OSIsoft, LLC
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Amandeep Singh Arora
+ */
+#include <sstream>
+#include <iostream>
+#include <reading.h>
+#include <wiringPi.h>
+#include <dht11.h>
+#include <logger.h>
+
+#define MAX_SENSOR_READ_RETRIES		(100)
+
+using namespace std;
+
+/**
+ * Constructor for the dht11 "sensor"
+ *
+ * @param pin	The RPi pin to which DHT11 sensor's data pin is attached (e.g. pin 7 = GPIO 4, this is the default)
+ */
+DHT11::DHT11(unsigned int pin) :
+m_pin(pin)
```

Comment:
why `m_pin(pin)` in new line?  I think we are not hitting max no of characters limit in 1 line